TILApp/Views/CommunityView/CommunityViewController.swift

```diff
@@ -1,22 +1,61 @@
 import UIKit
 
 final class CommunityViewController: UIViewController {
-    private lazy var label = {
-        let label = UILabel()
-        label.text = "CommunityViewController"
-        label.sizeToFit()
-        return label
-    }()
+    private let tableView = UITableView()
```

Comment:
```suggestion
    private lazy var tableView = UITableView().then {    		
        $0.dataSource = self
        $0.delegate = self
        $0.register(CommunityTableViewCell.self, forCellReuseIdentifier: "CommunityTableViewCell")
    }
```

이렇게 작성하면 더 깔끔할거 같아요 🙂